resources/dicts/conditions/pregnancy.json

```
@@ -16,7 +19,15 @@
     "All the old tales about queens fighting to the death to protect {PRONOUN/m_c/poss} babies suddenly make sense to m_c. {PRONOUN/m_c/subject/CAP} {VERB/m_c/feel/feels} up to fighting the sun itself, powered by the love {PRONOUN/m_c/subject} {VERB/m_c/feel/feels} for {PRONOUN/m_c/poss} yet-to-arrive litter.",
     "m_c feels like {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} eating enough at each meal to feed the whole Clan - {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} predicting a big litter. Or perhaps one very plump kit.",
     "m_c has begun hoarding little trinkets, stashing bits of fluff and feather and shiny smooth rocks into {PRONOUN/m_c/poss} birthing nest to stock it with kitten toys.",
-    "m_c invites {PRONOUN/m_c/poss} friends to feel {PRONOUN/m_c/poss} litter kick - surely these kittens are going to be talented warriors, because, well - ow."
+    "m_c invites {PRONOUN/m_c/poss} friends to feel {PRONOUN/m_c/poss} litter kick - surely these kittens are going to be talented warriors, because, well - ow.",
+    "m_c thinks {PRONOUN/m_c/subject}'ll have a small litter.",
+    "m_c hopes {PRONOUN/m_c/subject}'ll have a small litter. No way could {PRONOUN/m_c/subject} handle too many kits.",
+    "m_c thinks {PRONOUN/m_c/subject}'ll have a large litter.",
+    "m_c hopes {PRONOUN/m_c/subject}'ll have a large litter. m_c always wanted a big family after all.",
+    "m_c is unsure how many kits {PRONOUN/m_c/subject}'ll have, nor {VERB/m_c/do/does} {PRONOUN/m_c/subject} care. As long as they're happy and healthy, that's all that matters.",
+    "m_c dosen't know or care how many kits {PRONOUN/m_c/subject}'ll have.",
+    "m_c, citing mothers' intuition, claims {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject}'ll have a small litter.",
+    "m_c, citing mothers' intuition, claims {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject}'ll have a big litter."
```

"know" needs to be verb tagged

---

```suggestion
    "m_c, citing queen's intuition, claims {PRONOUN/m_c/subject} {VERB/m_c/know/knows}  {PRONOUN/m_c/subject}'ll have a big litter."
```
ditto to last suggestion